planet/subscription_request.py

```diff
@@ -45,12 +45,27 @@
 
 
 def build_request(name: str,
-                  source: dict,
-                  delivery: dict,
-                  notifications: Optional[dict] = None,
-                  tools: Optional[List[dict]] = None) -> dict:
+                  source: Mapping,
```

Comment:
Typing best practices (https://typing.readthedocs.io/en/latest/source/best_practices.html) is to accept mappings (most general) and return dicts (most specific). This is in accord with Postel's Law.